src/main/java/org/interledger/ilp/ledger/model/TransferStatus.java

```diff
@@ -28,5 +28,15 @@ public static TransferStatus valueOf(int status) {
         throw new IllegalArgumentException("Invalid TransferStatus status " + status);
     }
   }
+  
+  public static TransferStatus parse(String status) {
+      status = status.toLowerCase();
+      if ("proposed".equals(status) || "0".equals(status)) { return PROPOSED; }
```

Comment:
Should we be looking for the ordinal value here?  Seems like that should be invallid.